docs/Installation.rst

```diff
@@ -178,25 +121,142 @@ The following command should now mount the cgroup filesystem:
     /etc/init.d/cgroup start
 
 
-Python dependencies via pip
----------------------------
+Python dependencies
+===================
+
+These are all the python dependencies required to run CMS:
+
+.. literalinclude:: ../requirements.txt
+   :language: python
+
+These are all the python dependencies required to develop CMS:
+
+.. literalinclude:: ../dev-requirements.txt
+   :language: python
+
+There are good reasons to install Python dependencies via pip (Python Package Index) instead of your package manager, for example: two different Linux distributions may "offer" two different versions of ``python-sqlalchemy`` while, when using pip, you can choose to install a version that is known to be working correctly with CMS.
+
+The quick and dirty way of installing Python dependencies is this:
```

Comment:
To be fair, this is more of a pip problem. For example, npm allows to keep multiple versions of the same dependency in your home directory (so, in theory, you don't even need a virtual environment).